android/java/org/chromium/chrome/browser/preferences/website/BraveSiteSettingsPreferences.java

```diff
@@ -0,0 +1,39 @@
+/* Copyright (c) 2019 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.chromium.chrome.browser.preferences.website;
+
+import android.content.SharedPreferences;
+import android.os.Bundle;
+import android.support.v7.preference.Preference;
+
+import org.chromium.base.ContextUtils;
+import org.chromium.chrome.R;
+import org.chromium.chrome.browser.preferences.PreferenceUtils;
+
+public class BraveSiteSettingsPreferences extends SiteSettingsPreferences {
+    private static final String DESKTOP_MODE_CATEGORY_KEY = "desktop_mode_category";
+
+    @Override
+    public void onCreatePreferences(Bundle savedInstanceState, String rootKey) {
+        super.onCreatePreferences(savedInstanceState, rootKey);
+        PreferenceUtils.addPreferencesFromResource(this, R.xml.brave_site_settings_preferences);
```

Comment:
Because of this, preferences in `brave_site_settings_preference` will append to chromium's.
If we want to add our preferences in the middle of chromium's, we should modify `site_settings_preferences.xml` directly.